.buildkite/commands/lint.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash -u
 
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
```

Comment:
Out of my own curiosity - is it needed because of the release flavor and missing secrets, etc?

Reply:
Yes, precisle this 👍